.travis.yml

```diff
@@ -20,7 +20,7 @@ jobs:
   include:
     - stage: check
       script:
-        - npx aegir build --bundlesize
+        # - npx aegir build --bundlesize
```

Comment:
That might be good, it will just keep failing until we can remove all the non async modules. I originally just commented this out to avoid the 500 failures I was getting at the time from the bundlesize site.